format/Schema.fbs

```diff
@@ -183,10 +183,8 @@ table Decimal {
   /// Number of digits after the decimal point "."
   scale: int;
 
-  /// Number of bits per value. The only accepted width right now is 128 but
```

Comment:
Need to update the comments in lines 175-178 above ("Currently only 128-bit (16-byte) integers are used").

Reply:
done.